tests/transformations/subtests/transpile-js/webpack/package.json

```
@@ -4,11 +4,11 @@
     "build": "webpack -p"
   },
   "devDependencies": {
-    "webpack": "4.41.5",
+    "webpack": "^5.1.3",
     "webpack-cli": "3.3.10",
     "babel-loader": "8.0.6",
-    "@babel/core": "7.8.7",
-    "@babel/preset-env": "7.8.7",
-    "@babel/plugin-transform-runtime": "7.8.3"
+    "@babel/core": "7.11.6",
+    "@babel/preset-env": "7.11.5",
+    "@babel/plugin-transform-runtime": "7.11.5"
```

(to remove the `resolve.fullySpecified` workaround)

```suggestion
    "@babel/core": "7.12.10",
    "@babel/preset-env": "7.12.11",
    "@babel/plugin-transform-runtime": "7.12.10"
```